src/vos/evtree.c

```diff
@@ -170,7 +170,7 @@ time_cmp(uint64_t t1, uint64_t t2, int *out)
 		*out = RT_OVERLAP_OVER;
 	} else {
 		*out = RT_OVERLAP_UNDER;
-		if (t2 == EVT_REBUILD_MINOR_MIN) {
+		if (!exact && t2 == EVT_REBUILD_MINOR_MIN) {
```

Comment:
Though the epoch should be much larger than EVT_REBUILD_MINOR_MIN (when there is no unexpected bug), I think this check should be applied to minor epoch only.